src/collections/members/saurabh soni/index.mdx

```diff
@@ -0,0 +1,13 @@
+---
+name: Saurabh Soni
+position: Contributor
+image_path: ./saurabh.webp
+github: saurabh100ni
+twitter: https://twitter.com/Saurabh100nitwt
```

Comment:
- This link is not working in the preview @brijeshbhikadiya 
- Mention the username only instead of link to the profile 
> It is redirecting to `https://twitter.com/https://twitter.com/Saurabh100nitwt`

Reply:
Alright. I'll do that and revert to you